architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -0,0 +1,88 @@
+# base25 encoding for consignment references
+
+**Date:** 2020-11-13
+
+## Context
+
+As part of the 'transfer summary' page, the user will see a 'consignment reference' to identify their consignment (which can be useful if a user needs to contact us about a specific consignment). We came to the decision that we liked the format:
+
+`TDR-YEAR-base25reference`
+
+`TDR` - since the consignment will be transferred using the TDR service
+
+`YEAR` - year the consignment is created (not completed)
+
+`base25reference` - created once the consignment is assigned an incremental ID, which will be encoded to a base25 alpha-numeric code.
+
+Base25 was specifically chosen due to its ability to represent numeric identifiers through alpha-numeric codes. It also allows for specific letters and numbers to be removed from the list of possible characters, limiting the possibility that a reference can be created that is un-intentionally offensive.
+Base25 (specifically GCRb25) is also used within other TNA projects, and TDR utilising it will keep references to a similar standard to those used within different projects.
```

Comment:
Could you mention that we're not trying to match specific IDs used in those other systems? For example, we're not trying to generate the IDs for DRI or the Omega catalogue - they will continue to use their own ID generation.